testing/test_table.py

```diff
@@ -0,0 +1,40 @@
+from cached_property import cached_property
+
+from widgetastic.widget import Table
+from widgetastic.widget import View
+
+
+def test_table_cached_properties():
+    """Cached properties are defined as such or as properties with deleters"""
+    for item in Table._CACHED_PROPERTIES:
+        attribute = getattr(Table, item)
```

Comment:
as table_tree is not actually a cached property,
i would recommend to not have it in that particular list, but instead adding exactly one fitting del statement to the cache clearing method belonging to the class

that way all the special cases for it in the test are no longer needed, it needs however a few extra statements in the behavior test to consider that field

Reply:
Yeah, makes sense. I have refactored it. Looks better. Let me know your thoughts.

Reply:
well done :)